app/src/main/java/com/globant/practice/interfaces/Splash.java

```diff
@@ -0,0 +1,14 @@
+package com.globant.practice.interfaces;
+
+/**
+ * Work to decouple the view of the presenter.
+ * Created by jonathan.vargas on 31/03/2017.
+ */
+
+public interface Splash {
```

Comment:
If this Splash represents a view inside MVP, please place it into a **view** package. Also, it may have View as a suffix in the name.